contracts/periphery/oracle/src/contract.rs

```diff
@@ -3,6 +3,7 @@ use crate::querier::{query_cumulative_prices, query_pair_info, query_prices};
 use crate::state::{Config, PriceCumulativeLast, CONFIG, PRICE_LAST};
 use astroport::asset::{addr_validate_to_lower, Asset, AssetInfo};
 use astroport::oracle::{ExecuteMsg, InstantiateMsg, MigrateMsg, QueryMsg};
+use astroport::pair::PRICE_DECIMALS;
```

Comment:
Maybe let's rename to something more precise, like twap_precision?

Reply:
Renamed